File: helloworld/com/example/demo/NotabaristaRepository.java
package com.example.demo;

import com.example.demo.Notabarista;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.data.jpa.repository.JpaSpecificationExecutor;

public interface NotabaristaRepository extends JpaRepository<Notabarista, Integer>, JpaSpecificationExecutor<Notabarista> {

}

File: helloworld/src/main/kotlin/com/test/helloworld/NotabaristaRepository.kt
package com.test.helloworld

import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.data.jpa.repository.JpaSpecificationExecutor

interface NotabaristaRepository : JpaRepository<Notabarista?, Int?>, JpaSpecificationExecutor<Notabarista?> 